src/store/search/mutations.js

```diff
@@ -40,6 +40,20 @@ export default {
     state.latitude = null
     state.longitude = null
   },
+  [types.SET_MAP_CENTER] (state, { latitude, longitude }) {
+    state.mapCenterLatitude = latitude
+    state.mapCenterLongitude = longitude
+  },
+  [types.UNSET_MAP_CENTER] (state) {
+    state.mapCenterLatitude = null
+    state.mapCenterLongitude = null
+  },
```

Comment:
```

```
[types.SET_MAP_CENTER] (state, { latitude = null, longitude = null } = {}) {
    state.mapCenterLatitude = latitude
    state.mapCenterLongitude = longitude
  }
```

I think the two other mutations could also be merged with the above into a bigger `SEARCH__SET_MAP_OPTIONS` mutation.

Reply:
Right, I'll refactor it.